_control/info.php

```diff
@@ -0,0 +1,6 @@
+<?php
+  require_once __DIR__ . '/../_includes/autoload.php';
+  use Keyman\Site\Common\KeymanHosts;
+    // require_once('includes/template.php');
```

Comment:
Can this comment line be removed?

Reply:
Done - I kinda wonder about moving `/_control/info.php` into the https://github.com/keymanapp/shared-sites repo